core/server/services/mega/mega.js

```diff
@@ -128,6 +158,7 @@ const addEmail = async (postModel, options) => {
     const filterOptions = Object.assign({}, knexOptions, {limit: 1});
 
     const emailRecipientFilter = postModel.get('email_recipient_filter');
+    filterOptions.filter = transformEmailRecipientFilter(emailRecipientFilter);
 
     switch (emailRecipientFilter) {
```

Comment:
@kevinansfield Does that mean we can clean this up?

Reply:
Copy vs cut fail. Fixed!